docs/resources/mapreduce_cluster.md

```diff
@@ -445,6 +445,8 @@ The `nodes` block supports:
   -> This parameter is only valid and mandatory for `custom_nodes`.
 
 * `flavor` - (Required, String, ForceNew) Specifies the instance specifications for each nodes in node group.
+  You can obtain it on the cluster creation page in the console, but we recommend using browser developer tools to fetch
```

Comment:
pls add a data source to get flavor 
https://console.huaweicloud.com/apiexplorer/#/openapi/MRS/doc?version=v2&api=ShowMrsFlavors